tests/unhappypath/unhappypath_test.go

```diff
@@ -0,0 +1,309 @@
+package tests
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/sched-ops/k8s/stork"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/volume"
+	"github.com/portworx/torpedo/pkg/testrailuttils"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	storkops = stork.Instance()
+)
+
+const (
+	dropPercentage      = 20
+	delayInMilliseconds = 250
+	//24 hours
+	totalTimeInHours = 24
+	// TODO need to make it 60 minutes
+	errorPersistTimeInMinutes     = 10 * time.Minute
+	snapshotScheduleRetryInterval = 10 * time.Second
+	snapshotScheduleRetryTimeout  = 3 * time.Minute
+)
+
+func TestBasic(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_basic.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Basic", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+	// package installation hangs when using nsenter to run command on nodes
```

Comment:
This command will not be executed for nsenter so test should check if `tcpdump` utility is installed or not

Reply:
Added check for tcpdump availability, however, 
https://github.com/portworx/torpedo/blob/41744b4e0d8401f93fafb030eb0ebd97950b89af/drivers/node/ssh/ssh.go#L466
makes sure that it is not running on nsenter way.